### PR TITLE
fix: Update Twitter logo and name

### PR DESCRIPTION
**Pull request by Utkarsh3128**
added a new svg file for twiiter new logo and updated the code

Fix: #9096

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -5,14 +5,14 @@ markdown: kramdown
 kramdown:
   entity_output: as_input
 
-latest_leaflet_version: 1.9.4
+latest_leaflet_version: 1.9.2
```
Comment (WebSnke) [inline docs/_config.yml:8]: Can you explain why you downgraded the version?
This looks like a mistake to me but I could be wrong...

**Review by WebSnke — CHANGES_REQUESTED** [review]
I left some comments below.

I think that you pulled an old version of this repository. You should update to a current one to resolve the conflicts in the reference.html file.

File: docs/_config.yml (hunk 1 above)
Comment (Utkarsh3128) [reply docs/_config.yml:8]: I am really sorry for this, I didn't notice it.

Comment (WebSnke) [reply docs/_config.yml:8]: > I am really sorry for this, I didn't notice it.

You don't have to be sorry. Everyone makes mistakes.

**Comment by Hamjaster** [comment]
Is this issue resolved?

**Comment by WebSnke** [comment]
> Is this issue resolved?

What issue exactly?

File: docs/docs/images/new-twitter.svg
[the remark this answers is not in the document]
Comment (Navneetsinghnegi) [reply docs/docs/images/new-twitter.svg:1]: sorry, I did not understand. could you please explain this

**Comment by Falke-Design** [comment]
Thank you for your contribution but this is already fixed by #9102

**Pull request closed without merging**